docs/api/execute-test.md

```diff
@@ -5,41 +5,53 @@ parent: Benchmark API
 nav_order: 10
 ---
 
-A brief description of the command's function, and how it relates to the Benchmark workflow.
+The `execute_test` command is the primary usage of OpenSearch Benchmark. This command is used to execute a test.
 
 
 ## Syntax 
 
 ```bash
-execute_test
+opensearch-benchmark execute_test <args>
 ```
 
-// Table for any related fields. Example pulled from PPL page.
 
-Field | Description | Required
-:--- | :--- |:---
-`search` | Specify search keywords. | Yes
-`index` | Specify which index to query from. | No
-`bool-expression` | Specify an expression that evaluates to a boolean value. | No
+### Arguments
+
+**Note**: This is not an exhaustive list of arguments. 
```

Comment:
Any specific reason we're not able to make it exhaustive list?

Reply:
There's somewhere around 20 more arguments to document that deal with things like specific configurations for provisioning or reporting. I didn't think it was worth taking the time to add those here since they're used for very advanced use cases. 

I can add them in but it with take a little while

Reply:
I can add a section to the bottom for "Advanced Arguments" or something that lists them all

Reply:
These are already documented in the help text 🤦‍♂️ I'll update the table

Reply:
Decided to split the exhaustive list into a separate table to avoid burying the examples